sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/README.md

```diff
@@ -0,0 +1,17 @@
+﻿---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-service-bus
+name: Azure.Messaging.ServiceBus samples for .NET
+description: Samples for the Azure.Messaging.ServiceBus client library
+---
+
+# Azure.Messaging.ServiceBus Samples
+
+- [Send and receive messages from a queue](Sample01_HelloWorld.cs).
+- [Complete, abandon, defer, and deadletter messages](Sample02_MessageSettlement.cs).
+- [Send and receive session messages](Sample03_Sessions.cs)
+- [Use the processor to process messages which handles renewing message locks automatically](Sample04_Processor.cs)
```

Comment:
Can we provide one sample for schedule and cancel scheduled message?

Reply:
Added a sample, but leaving out of readme as I don't think it will be too commonly used.

Reply:
Yes, I was also suggesting the same :)